subsys/bluetooth/controller/Kconfig

```diff
@@ -497,6 +497,22 @@ config BT_CTLR_SYNC_PERIODIC
 config BT_CTLR_SYNC_PERIODIC
 	bool "LE Periodic Advertising in Synchronization State [EXPERIMENTAL]" if BT_LL_SW_SPLIT
 
+config BT_CTLR_SYNC_PERIODIC_ADV_LIST
```

Comment:
Isn't this config a redundant to the "CONFIG_BT_PER_ADV_SYNC_MAX"?

Reply:
No, that is an API parameter, and this is a Controller feature.